chaco/axis_view.py

```diff
@@ -32,11 +29,8 @@ def float_or_auto(val):
                     label="Main"),
                 Group(
                     Item("tick_color", label="Color", style="custom"),
-                         #editor=EnableRGBAColorEditor()),
```

Comment:
i'm happy when dead code is removed - but it'd be nice to mention this in the PR description.